x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_weblogs/ml/mean_bytes_by_file_extension.json

```diff
@@ -0,0 +1,38 @@
+
+{
+  "groups": [],
+  "description": "Kibana Sample Datasets Web Logs : detect anomalies in mean bytes",
```

Comment:
Should use sentence case, and for consistency with other modules - Kibana sample web logs data: Detect anomalies in mean bytes`